src/view/com/lightbox/ImageViewing/components/ImageItem/ImageItem.android.tsx

```diff
@@ -313,20 +314,16 @@ const ImageItem = ({
         singleTap,
       )
 
-  const isLoading = !isLoaded || !imageDimensions
   return (
     <Animated.View ref={containerRef} style={styles.container}>
-      {isLoading && (
-        <ActivityIndicator size="small" color="#FFF" style={styles.loading} />
```

Comment:
Couldn't figure out how to make this guy work well, but it seems less important now anyway. Figured I'd just remove it. Maybe we can readd with a bit more thinking.